MicrosoftFluentUI.podspec

```diff
@@ -181,6 +181,7 @@ fi', :execution_position => :before_compile }
     navigation_ios.platform = :ios
     navigation_ios.dependency 'MicrosoftFluentUI/ActivityIndicator_ios'
     navigation_ios.dependency 'MicrosoftFluentUI/Avatar_ios'
+    navigation_ios.dependency 'MicrosoftFluentUI/BadgeField_ios'
```

Comment:
I believe that similar to the other PR, using a generic view as the search bar leading view might make it more flexible and avoid this dependency on the BadgeField.

Reply:
Updated